resources/services/apps/daemon_sets.go

```diff
@@ -298,10 +298,10 @@ func DaemonSets() *schema.Table {
 // ====================================================================================================================
 
 func fetchDaemonSets(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
-	client := meta.(*client.Client).Services().DaemonSets
+	cl := meta.(*client.Client).Services().DaemonSets
```

Comment:
Shadowing.

We could also disable this rule, however it can prevent confusion when ready the code